homeassistant/components/sensor/filter.py

```diff
@@ -244,28 +249,36 @@ class OutlierFilter(Filter):
 
     Args:
         radius (float): band radius
+        median (float): band center
+            (defaults to NO_MANUAL_MEDIAN for automatic computation)
     """
 
-    def __init__(self, window_size, precision, entity, radius):
+    def __init__(self, window_size, precision, entity, radius,
+                 median=NO_MANUAL_MEDIAN):
         """Initialize Filter."""
         super().__init__(FILTER_NAME_OUTLIER, window_size, precision, entity)
         self._radius = radius
+        self._manual_median_enabled = (median == NO_MANUAL_MEDIAN)
```

Comment:
Why would you store this? I mean you can calculate it on the fly with `self._manual_median`.